### PR TITLE
Pal 1034 add gitattributes

### PR DESCRIPTION
**Pull request by pd104923**
- Added gitattributes file
- All script files have been updated to be executable
- All files have been normalised based on the new git configs so line endings have changed for some of them.
- All Dockerfiles have had any `sed` operations removed

### CODE REVIEW

**Review by m78233 — CHANGES_REQUESTED** [review]
Something odd with the shell files in the folder k8s-bash-scripts.  In github they are all tagged as empty files.  I look at them in my IDE, I can see the script.  I compare them to previous version and the message comes back that they are identical.  Could you have a close look at these in your environment?  See if you are having the same problem.

**Comment by pd104923** [comment]
This is not really an issue. I think they were marked as empty files in GitHub due to them being marked as executable. If you're seeing the script in the IDE and GitHub says they are the same then it's fine.

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]
A lot of files dont have any content changes, just line endings, please refactor this PR

**Comment by pd104923** [comment]
> A lot of files dont have any content changes, just line endings, please refactor this PR

This is due to the changes that were made for this ticket. When doing a `git add --renormalize .` for the repo these files had their line endings updated.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Set the default behavior, in case people don't have core.autocrlf set
```
Comment (developer01189998819991197253) [inline .gitattributes:1]: add palisade license header

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]
Update licenses in hadolint.yaml and .gitignore also

**Review by developer01189998819991197253 — APPROVED** [review]

**Pull request merged**